tests/helpers/testers.py

```diff
@@ -85,6 +85,7 @@ def _class_test(
     check_dist_sync_on_step: bool = True,
     check_batch: bool = True,
     atol: float = 1e-8,
+    extra_update_args: dict = None,
```

Comment:
```suggestion
    **kwargs_update: Any,
```

Reply:
I like this idea, It could also add support for positional arguments to the functions. The only drawbacks I see is that it would not be consistent with the metric_args argument and It would be slightly harder to read since having the separate explicit dictionary makes it perfectly clear that whatever is inside will be passed over to the update.

```
    _class_test(
        rank=0,
        worldsize=2,
        preds=torch.ones([10,1]),
        target=torch.ones([10,1]),
        metric_class=DummyMultiargMetric,
        sk_metric=fake_sk_metric,
        # extra_update_args={'z':torch.ones([10, 1])}  # < ----------- Option 1
        # z = torch.ones([10, 1]) # < --------------------------------------- Option 2
    )
```

I can definitely change it to either one.
On the other hand, I like the name "kwargs_update" a lot better than my original 'extra_update_args' so I might change my version to have that name (even if we decide not to change the behavior)

let me know what you think.

Reply:
Accepted in https://github.com/PyTorchLightning/metrics/pull/137/commits/1c591664be8c2b631477f97fd654c66b121891a1